changelog/v1.19.0-beta3/issue_10445.yaml

```diff
@@ -0,0 +1,6 @@
+changelog:
+  - type: FIX
```

Comment:
WYDT about the NON_USER_FACING type here? The changes here help automate some manual toil during GW API bumps and shouldn't have any user-facing implications.

Reply:
Ack. Commit https://github.com/solo-io/gloo/commit/a669ad3ad076c526a1c7d614db31551939414520 changes the changelog type to `NON_USER_FACING`.

Reply:
fwiw i'm still seeing `FIX`, maybe the change wasn't pushed?